[Tidy] Prepare for Vizro-AI release (#651)
mckinsey/vizro: vizro-ai/docs/pages/tutorials/quickstart-dashboard.md

# Dashboard generation

> !!! warning "Alpha Feature"

Reviewer: ```suggestion
!!! warning "This feature is under development"
```

Reviewer: Or something similar I think, rather than "Alpha"

Author: To be more precise, it's not "under development" anymore. 😆 @maxschulz-COL wdyt?

Reviewer 2: I would stick with "alpha" or maybe something like - "Feature currently in alpha version", because what is an alpha feature 😂 like a very dominant feature??

>     This feature is currently in **Alpha**. It is stable but not officially released yet. We welcome feedback via the [GitHub Issues](https://github.com/mckinsey/vizro/issues), and be aware that it may still undergo minor adjustments.

Reviewer 2: ```suggestion
    This feature is currently in **Alpha**. It is stable but not officially released yet. We welcome feedback via [GitHub Issues](https://github.com/mckinsey/vizro/issues). Please note that the feature and its API may still undergo minor adjustments.
```
